README.md

```diff
@@ -35,46 +35,53 @@ This is a vue-cli 3.x plugin to add an Node Express server in your Vue project.
 
 :warning: Make sure you have vue-cli 3.x.x:
 
-```
+```sh
 vue --version
 ```
 
-If you don't have a project created with vue-cli 3.x yet:
+If you have not installed vue-cli 3, first follow the install instructions here: [https://cli.vuejs.org/](https://cli.vuejs.org/)
 
-```
+Generate a project using vue-cli 3
+
+```sh
 vue create my-new-app
 ```
 
-Navigate to the newly created project folder and add the cli plugin:
+Navigate to the newly created project folder and add this plugin. It is good practice to commit your changes prior to installing a new plugin.
 
-```
+```sh
 cd my-new-app
 vue add express
 ```
-Soon:
-*:information_source: An example `APIExample.vue` component will be added into your sources if you chose to include the examples.*
+
+:soon: _:information_source: An example `APIExample.vue` component will be added should you choose to include examples when installing the plugin._
 
 ## Usage
 
-To start your server for development purpose, use this commands:
+To start your server for development, run:
 
-```
+```sh
 yarn express
 ```
 
-The server will be automatically restarted when a change is detected.
+The server will watch for file changes and automatically restart.
 
-You just then have to start the app:
+Then you just have to start the web app:
 
-```
+```sh
 yarn serve
 ```
 
-To run the server only once for production use, run:
-```
-yarn express:run
+Alternatively you can serve the web app and api together from express by setting the plugin option, `shouldServeApp` to `true`. See the [Configuration](#configuration) section below for further details. You will also need to tell Express where to serve your web application by adding the following to `./srv/index.js`;
+
+```js
+var router = express.Router();
+router.use(express.static("./dist"));
+app.use("/", router);
```

Comment:
All the idea of this package is to avoid that (also, this will not work with vue-router)! `shouldServeApp` to true (chich is the default) is sufficient to serve properly the app.

Could you remove this part please?

Reply:
I added this as I found it necessary to serve my front-end code. I have also got vue-router working with it.
In my project, 'shouldServeApp` is set to true but when I run using `yarn express`  an error appears on the screen get _no routes available_ and the server doesn't start.
I'm likely doing some thing wrong as

Reply:
I have removed the code example